pep/app/constants/search.ts

```diff
@@ -29,6 +29,50 @@ export enum ViewPeriod {
     PAST_12_MONTHS = 4
 }
 
+export enum FacetId {
+    art_id = 'art_id',
```

Comment:
@billdami normally I believe we capitalize the left side i.e. ART_ID to make it more clear its a enum/constant